dcos/config.py

```diff
@@ -43,7 +43,16 @@ def set_val(name, value):
 
     save(toml_config)
 
-    if not value_exists:
+    if name == "core.dcos_acs_token":
+        msg = "[{}]: ".format(name)
+        if not value_exists:
+            msg += "set"
+        elif old_value == new_value:
+            msg += "already set to that value"
+        else:
+            msg += "changed"
+        emitter.publish(msg)
+    elif not value_exists:
```

Comment:
yes, we print it when we change the acs_token from a request to auth: https://github.com/mesosphere/dcos-cli/blob/master/dcos/http.py#L440